docs/usage/getting-started/running.md

```diff
@@ -92,8 +95,15 @@ It is built similarly to the default "full" Renovate image described above, but
 - It is released every 1-2 months in a slower, more stable cadence than Renovate OSS, which releases on every commit
 - It's licensed using an end-user license agreement (EULA) and not the Affero General Public License (AGPL)
 
-WSOP supports GitHub (both `github.com` and GitHub Enterprise Server) as well as GitLab self-hosted.
-Documentation can be found in its public GitHub repository [`whitesource/renovate-on-prem`](https://github.com/whitesource/renovate-on-prem).
+Plus, the Enterprise Edition has:
+
+- Horizontal scaling to run multiple 'worker' containers
+- Dedicated support from Mend.io
+- Premium features, including Smart Merge Control
+
+Note: Visit the Mend.io website to learn more about [Renovate Enterprise Edition](https://www.mend.io/renovate-enterprise/).
+
+For help configuring Renovate CE or Renovate EE, refer to the documentation on the public GitHub repository [`mend/renovate-ce-ee`](https://github.com/mend/renovate-ce-ee).
```

Comment:
```suggestion
To learn how to configure Renovate CE or Renovate EE, read the documentation on the public GitHub repository [`mend/renovate-ce-ee`](https://github.com/mend/renovate-ce-ee).
```

`For help ...` implies that you'll get help from us. But what we really mean is that the reader should go read the docs. So we should say that instead.